.github/workflows/validate_deps.yml

```diff
@@ -6,8 +6,7 @@ on:
     branches: [main]
     types: [opened, synchronize]
 
-permissions:
-  id-token: write
+permissions: {}
```

Comment:
You might need read-all?

Reply:
It should work without that, since it's just reading code, doesn't need the token